pkg/daemon/ceph/multus/utils.go

```diff
@@ -0,0 +1,286 @@
+package multus
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/coreos/pkg/capnslog"
+	"github.com/vishvananda/netlink"
+	"gopkg.in/yaml.v2"
```

Comment:
try to use "k8s.io/apimachinery/pkg/util/yaml", we are trying to remove YAML deps to only use one